template-parts/site-navigation/wrapper.php

```diff
@@ -4,11 +4,11 @@
  *
  * @package shiro
  */
+
 ?>
 <nav class="primary-nav">
 	<div class="primary-nav__drawer" data-dropdown-content="primary-nav">
 		<?php
-		get_template_part( 'template-parts/site-navigation/search' );
```

Comment:
Removing this template part means there's no way to search the site on mobile. I wonder if it should be left in and just visually hidden for desktop views?